versioned_docs/version-4.0.0-alpha.1/macros-attributes/chain-extension.md

```diff
@@ -95,7 +89,7 @@ pub trait MyChainExtension {
 
 Default value: `true`
 
-By default all chain extension methods return a `Result<T, E>` where `E: From<Self::ErrorCode>`.
+By default all chain extension methods should return a `Result<T, E>` where `E: From<Self::ErrorCode>`.
```

Comment:
Can you clarify what the implication is if they don't?

Reply:
I think lines 106-107 states the implication

Reply:
> Can you clarify what the implication is if they don't?

I think lines 106-107 states the implication